src/pyproject.toml.m4

```diff
@@ -21,3 +21,8 @@ requires = [
         memory_allocator \
                     ')]
 build-backend = "setuptools.build_meta"
+
+[tool.conda-lock]
+platforms = [
+    'osx-64', 'linux-64', 'linux-aarch64', 'osx-arm64'
+]
```

Comment:
OK, the config for aarch/arm is here but shouldn't the resulting files be committed too?

Reply:
Yes, I would prefer to do this as part of the follow-up that adds the automatic update. It's a bit of tedious work to create all of them manually.

Reply:
How about adding a maintenance script that makes the necessary calls. 
Then gh action

Reply:
Done.